tensorflow/core/platform/default/logging.cc

```diff
@@ -109,14 +109,29 @@ int64 MinLogLevel() {
   }
 }
 
+int64 MinLogLevelFromEnv() {
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_LOG_LEVEL");
+  return LogLevelStrToInt(tf_env_var_val);
+}
+
+int64 MinVLogLevelFromEnv() {
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_VLOG_LEVEL");
```

Comment:
That looks fine too -- let's run a test to see if it passes on all configurations.